### PR TITLE
Fix some issues with automatic release publication

### PR DESCRIPTION
**Pull request by Takishima**

### CODE REVIEW

File: .github/workflows/publish_release.yml [hunk 1]
```diff
@@ -14,78 +15,69 @@ jobs:
...
     name: Build wheels on ${{ matrix.os }}
     runs-on: ${{ matrix.os }}
-    if: startsWith(github.ref, 'refs/tags/') || (github.event_name == 'pull_request' && github.event.pull_request.merged == true)
+    if: startsWith(github.ref, 'refs/tags/') || (github.event_name == 'pull_request' && github.event.pull_request.merged == true) || github.event_name == 'workflow_dispatch'
     strategy:
       matrix:
-        os: [ubuntu-20.04, windows-2019, macos-10.15]
+        os:
+          - ubuntu-latest
+          # - windows-latest
```
Comment (andreashehn) [inline .github/workflows/publish_release.yml:23]: Please remove commented code, otherwise it might be forgotten and will live on forever.
If you need it again, you can always get it from the git history.

File: CHANGELOG.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 -   Fix GitHub workflow for publishing a new release
 
+
```
Comment (andreashehn) [inline CHANGELOG.md:23]: unnecessary change?

**Review by andreashehn — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Takishima) [reply CHANGELOG.md:23]: Unnecessary, yes. But I do not want to write twice the same thing in the changelog to make the newly instated CI check pass…

Comment (andreashehn) [reply CHANGELOG.md:23]: ok, makes sense.

File: .github/workflows/publish_release.yml (hunk 1 above)
Comment (Takishima) [reply .github/workflows/publish_release.yml]: Done.

**Review by andreashehn — APPROVED** [review]
